pkg/common/utils/chunk_test.go

```diff
@@ -35,6 +35,33 @@ func TestChunkParseChunkSizeGetCorrect(t *testing.T) {
 	}
 }
 
+func TestChunkParseChunkSizeGetError(t *testing.T) {
+	// test err from -----n, err := bytesconv.ReadHexInt(r)-----
+	chunkSizeBody := ""
+	zr := mock.NewZeroCopyReader(chunkSizeBody)
+	chunkSize, err := ParseChunkSize(zr)
+	assert.DeepEqual(t, true, err != nil)
```

Comment:
better use "assert.NotNil()" instead of this

Reply:
the same goes for the following

Reply:
i have already modify them